lib/Onyx.ts

```diff
@@ -307,8 +307,10 @@ function merge<TKey extends OnyxKey>(key: TKey, changes: OnyxMergeInput<TKey>):
         }
 
         try {
-            // We first only merge the changes, so we can provide these to the native implementation (SQLite uses only delta changes in "JSON_PATCH" to merge)
```

Comment:
A part of me is confused about why we are getting rid of JSON_PATCH. Another part believes that this was a _benefit_ since when we do this we end up passing a smaller delta change instead of a potentially large object and therefore it should be more efficient (hand wave there as I do not entirely recall anymore).